tests/utils/pvc.go

```diff
@@ -15,18 +15,19 @@ import (
 
 const (
 	DefaultPvcMountPath = "/pvc"
-	pvcPollInterval     = 2 * time.Second
-	PVCCreateTime       = 30 * time.Second
-	PVCDeleteTime       = 30 * time.Second
-	PVCPhaseTime        = 30 * time.Second
+	defaultPollInterval = 2 * time.Second
```

Comment:
I'm ok defining a "catch-all" poll interval and that constant can contain _default_ in its name. But, when the const is used for a specific resource I'd prefer to see that resource name in the const name. What was the motivation to change this?

Reply:
Are you saying define `const defaultPollInterval` then const `PVCPollInterval = defaultPollInterval`?